### PR TITLE
Add final updates

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
# Add final updates

In this PR, software gardening and general updates are performed to prepare the repo for minting. Specifically:

1. Remove plate 5 from metadata as it was not used for any analysis
2. Update README to be more related to the manuscript and update paths
3. Check envs for reproducibility (except for the CP env which is hard for us to control for reproducibility, as we know lol)

Please confirm that the README makes sense/clear and if I missed anything. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,55 +23,53 @@ In this modified Cell Painting, there are five channels:
...
+Wells are labelled as either "Failing" (failing heart) or "Healthy" (non-failing heart).
+There are three different treatments:
 
-![localhost220513100001_KK22-05-198_FactinAdjusted_platemap_figure.png](./metadata/platemap_figures/localhost220513100001_KK22-05-198_FactinAdjusted_platemap_figure.png)
+- DMSO (control)
+- drug_x
+- TGRFi (TGFB inhibitor)
 
-For the third plate, we are using the following plate design:
+We applied the model to this place to assess that the model is able to accurately predict the control cells and how the model predictions change with the treatments.
```
Comment (axiomcura) [inline README.md:47]:
```suggestion
We applied the model to this dataset to evaluate its accuracy in predicting control cells and to observe how its predictions change in response to different treatments.
```

Feel free to reject if my edits generates a different message you're trying to convey.

File: README.md [hunk 2]
```diff
@@ -14,55 +23,53 @@ In this modified Cell Painting, there are five channels:
...
 
 - **localhost230405150001**
 
 ![localhost230405150001_platemap_figure.png](./metadata/platemap_figures/localhost230405150001_platemap_figure.png)
 
-In this plate, there are only two different patients, one with a healthy heart and one that had a failing heart. 
 
-For the fourth plate, we used the following plate design:
+Additionally, we include pilot plates in this repository that did not used an optimized protocol for extra analysis that we do not include in the manuscript.
```
Comment (axiomcura) [inline README.md:54]:
```suggestion
Additionally, we include pilot plates in this repository that were not prepared using an optimized protocol, intended for further analysis that is not included in the manuscript.
```

**Review by axiomcura — APPROVED** [review]
LGTM! Left some suggestions c: Great job!

File: README.md (hunk 1 above)
Comment (jenna-tomkinson) [reply README.md:47]: This work really well, updated!

File: README.md (hunk 2 above)
Comment (jenna-tomkinson) [reply README.md:54]: I like this change, added!

**Comment by jenna-tomkinson** [comment]
@axiomcura Thank you lots for the review!

@gwaybio Feel free to give this a final look over before merging!

**Review by gwaybio — APPROVED** [review]

**Comment by jenna-tomkinson** [comment]
Thank you both! I am merging now!

**Pull request merged**